Core/include/Acts/Seeding/detail/StrawLineFitAuxiliaries.hpp

```diff
@@ -0,0 +1,203 @@
+  ///        cannot be updated due to parallelism between straw wire and line
+  /// @param line: Reference to the line to which the residual is calculated
+  /// @param hitMinSeg: Difference of the line reference point & the straw
+  ///                   position
+  /// @param wireDir: Direction vector of the straw wire
+  /// @param driftRadius: Measured radius of the straw measurement
+  bool updateStrawResidual(const Line_t& line, const Vector& hitMinSeg,
+                           const Vector& wireDir, const double driftRadius);
+  /// @brief Calculates the along-the wire component of the straw
+  ///        measurement's residual to the line
+  /// @param line: Reference to the line to which the residual is calculated
+  /// @param hitMinSeg: Difference of the line reference point & the straw
+  ///                   position
+  /// @param wireDir: Direction vector of the straw wire
+  void updateAlongTheStraw(const Line_t& line, const Vector& hitMinSeg,
+                           const Vector& wireDir);
+  /// @brief Calculates the residual of a strip measurement w.r.t. the line
+  /// @param line: Reference to the line to which the residual is calculated
+  /// @param normal: Reference to the vector normal on the strip measurement plane
+  /// @param sensorN: Reference to the first basis vector inside the strip measruement plane,
+  ///            which is given by the sensor normal
+  /// @param sensorD: Reference to the second basis vector inside the strip measruement plane,
+  ///            which is given by the sensor direction
+  /// @param stripPos: Position of the strip measurement
+  /// @param isBending: Flag toggling whether the precision direction is constrained
+  /// @param isNonBending: Flag toggling whether the non-precision direction is constrained
+  void updateStripResidual(const Line_t& line, const Vector& normal,
+                           const Vector& sensorN, const Vector& sensorD,
+                           const Vector& stripPos, const bool isBending,
+                           const bool isNonBending);
+  /// @brief Resets the residual and all partial derivatives to zero.
+  void reset();
+  Config m_cfg{};
+  std::unique_ptr<const Logger> m_logger{};
+
+  /// @brief Cached residual vector calculated from the measurement & the parametrized line
+  Vector m_residual{Vector::Zero()};
+  /// @brief Partial derivatives of the residual w.r.t. the fit parameters parameters
+  std::array<Vector3, FitParIndices::nPars> m_gradient{
+      filledArray<Vector3, FitParIndices::nPars>(Vector3::Zero())};
+  /// @brief  Second partial derivatives of the residual w.r.t. the fit parameters parameters
+  static constexpr std::size_t s_nPars = FitParIndices::nPars;
+  std::array<Vector3, sumUpToN(s_nPars)> m_hessian{
+      filledArray<Vector3, sumUpToN(s_nPars)>(Vector3::Zero())};
+
+  ///
+  ///  Auxiliary parameters needed to calculate the residual of the
+  ///  point of closest approach and its derivatives
```

Comment:
```suggestion
  //  Auxiliary parameters needed to calculate the residual of the
  //  point of closest approach and its derivatives
```
since this is a non-docstring